ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -106,32 +106,52 @@ else
 		then
 		fi
 	else
-		# FMUv1
-		if mpu6000 start
+		if ver hwcmp MINDPX_V2
 		then
-		fi
+			if mpu6500 start
+			then
+			fi
 
-		if l3gd20 start
-		then
-		fi
+			if lsm303d start
+			then
+			fi
 
-		# MAG selection
-		if param compare SENS_EXT_MAG 2
-		then
-			if hmc5883 -C -I start
+			if l3gd20 start
+			then
+			fi
+
+			# External I2C bus
+			if hmc5883 -C -T -X start
 			then
 			fi
 		else
```

Comment:
Please change this instead to:

```
if ver hwcmp MINDPX_V2
then
fi

if ver hwcmp PX4FMU_V2
then
fi

if ver hwcmp PX4FMU_V1
then
fi
```

The current if / else nesting is unreadable. When you do this the diff should also become readable again.